content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/1-mysql.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "MySQL"
+metaTitle: "Palette Dev Engine MySQL Service"
+metaDescription: "Palette Dev Engine MySQL Service"
+hideToC: false
+type: "appTier"
+category: ['databases']
+fullWidth: false
+logoUrl: "https://registry.dev.spectrocloud.com/v1/mysql-operator/blobs/sha256:2d59bc428916752528280eac03330d712164163e2f3c476409f5c25d8a7c2778?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# MySQL
+
+[MySQL](https://www.mysql.com/) is an open-source relational database management system commonly used in web applications and other software that requires a database. It is known for its reliability, ease of use, and flexibility. MySQL is covered under the GNU license and uses structured query language (SQL) to manage data with the following properties:
+
+<br />
+
+* Creates a database for storing and manipulating data and defining the relationship of each table.
+
+
+* Clients can retrieve and manipulate data by creating SQL queries and submitting them to the MySQL instance.
+
+
+# Add MySQL to App Profile
+
+Use the following steps to add MySQL to an app profile.
+
+<br />
+
+## Prerequisite
+
+A Spectro Cloud [account](https://www.spectrocloud.com/get-started/).
+
+<br />
+
+## Enablement
+
+1. Log in to [Palette](console.spectrocloud.com).
+
+
+2. On the right side of the window, click on the **User Menu**. Once the user menu is expanded, click on **Switch to App Mode**.
+
+
+3. Navigate to the left **Main Menu** and click on **App Profiles** to create a [new App Profile](/devx/app-profile/create-app-profile/). Provide the following basic information and click **Next**.
+
+|    **Parameter**            | **Description** |
+|-----------------------------|-----------------|
+|Application Profile Name     | A custom name for the app profile.|
+|Version (optional)           | The default value is 1.0.0. You can create multiple versions of an App Profile using the format **`major.minor.patch`**.|
+|Description (optional)       | Description of the app profile.| 
+|Tag (optional)               | Assign tags to the app profile.|
+ 
+
+4. Select **MySQL** from the database services and start the configuration.
+  
+
+5. Provide the following information to the wizard:
+  * **Name:** The database name. You can use the auto generated name or create a custom name.
```

Comment:
```suggestion
  * **Name:** The database name. You can use the auto-generated name or create a custom name.
```

Reply:
In one of the other files, I think it said Palette-generated. Auto-generated is preferable. I'm not sure which file it was in to change it to 'auto-generated'.